--- a/modules/db.postgresql/db_postgresql_re_grt.py
+++ b/modules/db_postgresql/db_postgresql_re_grt.py
@@ -74,5 +74,6 @@

-        seq_details_query = """SELECT min_value, max_value, start_value, 
-increment_by, last_value, is_cycled, cache_value
-FROM "%s"."%s" """
+        seq_details_query = """SELECT min_value, max_value, start_value, 
+increment_by, last_value, cycle as is_cycled, cache_size as cache_value
+FROM pg_catalog.pg_sequences
+WHERE schemaname = '%s' AND sequencename = '%s' """


